app/main/helpers/search_helpers.py

```diff
@@ -20,6 +22,33 @@ def get_page_from_request(request):
         return None
 
 
+def get_request_url_without_any_filters(request, filters):
+    '''
+        Logic:
+            -> get all url args
+            -> loop through the filters list and pop any matching url args
+            -> build and return url path
+
+        Arguements:
+            -> request - Request Object (import from flask)
+            -> filters - list of all filters from digital market place framework
+    '''
```

Comment:
A few Python style issues here:
1) Docstrings for methods should use triple double-quotes rather than single-quotes.
2) Don't really need to document the logic; we can work that out from your implementation. It's good to document why rather than how.
3) Typo in spelling of 'arguments'
4) It would be good to use a recognised docstring format, whether it's reST or Google-esque. See https://stackoverflow.com/questions/3898572/what-is-the-standard-python-docstring-format. (I prefer Google-esque... 🙂 ).